models/customer.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from datetime import datetime
+
+class Customer:
```

Comment:
You really don't need 3 classes here to do 1 job.  One class to handle interacting with the API would be good.